components/modal/modal.css

```diff
@@ -0,0 +1,26 @@
+.modal-container {
```

Comment:
you can re-use "start-container" here. The properties for both are identical

Reply:
for making code more understandable,readable and easy to implement i have made a separate class